JUnit version 4.13.2
......................................................................................................................................................................................................................................................E.........E...................................
Time: 14.301
There were 2 failures:
1) test061(org.flywaydb.core.api.configuration.ClassicConfiguration_ESTest)
java.lang.AssertionError: Exception was not thrown in org.flywaydb.core.internal.logging.buffered.BufferedLog but in org.flywaydb.core.internal.logging.EvolvingLog.debug(EvolvingLog.java:52): java.lang.NullPointerException
	at org.evosuite.runtime.EvoAssertions.assertThrownBy(EvoAssertions.java:111)
	at org.evosuite.runtime.EvoAssertions.verifyException(EvoAssertions.java:48)
	at org.flywaydb.core.api.configuration.ClassicConfiguration_ESTest.test061(ClassicConfiguration_ESTest.java:756)
2) test058(org.flywaydb.core.api.configuration.ClassicConfiguration_ESTest)
java.lang.AssertionError: Exception was not thrown in org.flywaydb.core.internal.logging.buffered.BufferedLog but in org.flywaydb.core.internal.logging.EvolvingLog.debug(EvolvingLog.java:52): java.lang.NullPointerException
	at org.evosuite.runtime.EvoAssertions.assertThrownBy(EvoAssertions.java:111)
	at org.evosuite.runtime.EvoAssertions.verifyException(EvoAssertions.java:48)
	at org.flywaydb.core.api.configuration.ClassicConfiguration_ESTest.test058(ClassicConfiguration_ESTest.java:703)

FAILURES!!!
Tests run: 290,  Failures: 2

